### PR TITLE
Product detail page

### PR DESCRIPTION
**Pull request by malikmasis**
closes https://github.com/abpframework/abp/issues/11428

### CODE REVIEW

**Review by enisn — APPROVED** [review]

File: apps/public-web/src/EShopOnAbp.PublicWeb/Pages/ProductDetail.cshtml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+﻿@page "{id:guid}"
```
Comment (enisn) [inline apps/public-web/src/EShopOnAbp.PublicWeb/Pages/ProductDetail.cshtml:1]: how about making url `/products/{id}` ?

**Comment by malikmasis** [comment]
image

**Comment by malikmasis** [comment]
image

File: apps/public-web/src/EShopOnAbp.PublicWeb/Pages/ProductDetail.cshtml (hunk 1 above)
Comment (malikmasis) [reply apps/public-web/src/EShopOnAbp.PublicWeb/Pages/ProductDetail.cshtml:1]: I follow the same approach with OrderReceived
```
@page "{orderNo:int}"
```

**Comment by malikmasis** [comment]
image

**Review by enisn — APPROVED** [review]

**Pull request merged**